static/sass/_pattern_l-fluid-breakout.scss

```diff
@@ -0,0 +1,106 @@
+@mixin l-charmhub-fluid-breakout {
+  .l-fluid-breakout {
+    .l-fluid-breakout__aside {
+      .p-muted-heading,
+      .p-side-navigation__list {
+        @media (max-width: $breakpoint-large - 1px) {
+          padding-inline-start: 1rem;
```

Comment:
what is this for?

Reply:
this is to add padding to the left for the filters (checkboxes). Otherwise they'll get to the edge of the screen.

Reply:
to match the grid margin-left? If so I would suggest map-get($grid-margin-widths, small); this way if we change the grid margins, this will stay in sync with it.

Reply:
ok